sessions/js-array-methods-2/zoo-animals/utils/db.js

```diff
@@ -0,0 +1,103 @@
+export const zooAnimals = [
+    "African Wild Ass",
```

Comment:
Don't get me wrong I absolutely love savannah donkeys, but I suggest to start from entry two of this list!

Reply:
😂😂😂😂 I remove it 👍